### DISCUSSION TITLE
Creating and calling actions within ViewColumn

### DISCUSSION THREAD

**Discussion by gsusanj** [filamentphp/filament#10944] [Help]
Hi,
I have created a Resource with ViewColumn, withing which I am trying to render Filament\Actions\Action buttons.
The buttons gets rendered, using the URL it opens correctly, but whenever modals are included, it does not work.

Both, ->form() and ->requiresConfirmation() fail silently. The loading icon shows up, the request is ran and the component is reloaded, but the modal does not show up and no error is thrown anywhere.

I have included <x-filament-actions::modals /> in the ViewColumn view.

**Comment by leandrocfe** [comment]
Could you share the whole code please?
